2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,12 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let report = [];
+    for(element of cities){
```

Comment:
In order to properly declare the loop variable, you should use let to declare element.